src/site/filters/liquid.js

```diff
@@ -12,6 +12,10 @@ function getPath(obj) {
 module.exports = function registerFilters() {
   const { cmsFeatureFlags } = global;
 
+  // Set timeout option to something higher (20mins)
+  // eslint-disable-next-line no-new
+  new liquid.Context({ timeout: 1200000 });
```

Comment:
Do you think this will be beneficial to master?

Reply:
@jhonnyoddball nothing in the current state will, but it's more so looking down the line and noting the errors we saw, and planning for them now